src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/Razor/ComponentAccessibilityCodeActionProviderTest.cs

```diff
@@ -169,6 +169,33 @@ public async Task Handle_NewComponent_SupportsFileCreationTrue_ReturnsResult()
         Assert.NotNull(command.Data);
     }
 
+    [Fact]
+    public async Task Handle_NewComponent_CaretInAttribute_ReturnsResult()
```

Comment:
I'm confused by this title, it says `CaretInAttribute` but then there is no Caret in the attribute on line 177?

Reply:
This doesn't use the `$$` syntax (though it should, as per Allison's comment), the `23` on line 184 puts the caret in the attribute.